docs(readme): edit (#1)
algolia-samples/email-recommendations: README.md

# Email recommendations

This sample app shows how to leverage Algolia Search and Algolia Recommend to display product recommendations in emails.
Showing recommendations is a great way to engage your customers at various stages of their journey before and after visiting your site.

This sample app comes with the following recommendation models:

> - A customer browsed in a category: recommend **best rated** products from that category. This recommendation model uses Algolia's [Faceting](/doc/guides/managing-results/refine-results/faceting/) feature.

Reviewer: ```suggestion
- A customer browsed a category: recommend **best rated** products from that category. This recommendation model uses Algolia's [Faceting](https://www.algolia.com/doc/guides/managing-results/refine-results/faceting/) feature.
```

> - A customer just bought a product: recommend products that are **frequently bought together**. This recommendation model uses [Algolia Recommend](/doc/guides/algolia-ai/recommend/) to train a machine-learning algorithm based on which products users often buy together.

Reviewer: ```suggestion
- A customer just bought a product: recommend products that are **frequently bought together**. This recommendation model uses [Algolia Recommend](https://www.algolia.com/doc/guides/algolia-ai/recommend/) to train a machine-learning algorithm based on which products users often buy together.
```

> - A customer just bought a product: recommend **related products**. This model also leverages Algolia Recommend to train a machine-learning algorithm to find products that similar.

Reviewer: ```suggestion
- A customer just bought a product: recommend **related products**. This model also leverages Algolia Recommend to train a machine-learning algorithm to find products that are similar.
```

> - A customer has an existing user profile: recommend products that match their _affinities_. This model uses Algolia Recommend and [Personalization](/doc/guides/personalization/what-is-personalization/).

Reviewer: ```suggestion
- A customer has an existing user profile: recommend products that match their _affinities_. This model uses Algolia Recommend and [Personalization](https://www.algolia.com/doc/guides/personalization/what-is-personalization/).
```

## Features

This sample app comes with the following features:

- A rock-solid base email template with [Cerberus](https://tedgoas.github.io/Cerberus/)
- A rich and powerful templating language with [Nunjucks](https://mozilla.github.io/nunjucks/)
- Three different email templates for different moments of the customer's journey (pre-order, post-order, re-engagement)
- Four different models for recommended products with Algolia Recommend and faceting

## Demo (Try it yourself!)

> [Run the demo](https://d8nd8.sse.codesandbox.io/)

Reviewer: ```suggestion
[Run the demo](https://emails-recommendations.herokuapp.com/)
```
